dev/cli

```diff
@@ -31,32 +31,48 @@ GLOBAL OPTIONS
 COMMANDS
     --authn-oidc    Enables OIDC features
     --authn-azure   Enables Azure features
-    --authn-gce     Enables GCE features
+    --authn-gce [GCE_INSTANCE_NAME] Enables GCE features passing in the Google Compute Engine VM name
 EOF
 exit
 }
 
 function enable_gce() {
+  local gce_instance_name="$1"
+  local gce_zone="$2"
+
+  if [ -z "$gce_instance_name" ]; then
+    local prgname=$(basename "$0")
+    echo "Missing GCE instance name. Usage ./$prgname exec --authn-gce [GCE_INSTANCE_NAME] --zone=[GCE_ZONE], \
+(--zone is optional: default zone is 'us-central1-a')"
```

Comment:
why is there a default zone? why is it this one?

Reply:
Resources in GCP such as virtual machine instances that live in a zone, one must specify the zone in which the vm is order to execute a command otherwise the default will be [us-central-1].
```
gcloud compute instances describe my-us-vm-test
No zone specified. Using zone [us-central1-a] for instance: [cucumber-test].
```
`my-us-vm-test` is located in [us-central-1], so `gcloud` will find it.